sql-cli/src/sql_cli/utils.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
```

Comment:
In future, we may be able to rename this module to something jinja related - so we avoid generic utils blowing up. I'm happy for us to do this in a separate PR as well :)